### DISCUSSION TITLE
i need wallpaper from hyperdots preview. Could any one give me a file?

### DISCUSSION THREAD

**Discussion by Sofitt** [prasanthrangan/hyprdots#1993] [General]
https://raw.githubusercontent.com/prasanthrangan/hyprdots/main/Source/assets/rofi_style_12.png

I cant install theme with script, because my main system is Ubuntu. So i just need image

**Comment by tobin-rieger-cameron** [comment]
Do you know what theme this is from? 
did some hunting in my files and couldn't find that wallpaper. 

You might just have to pull directly from the github if you're not able to run the script

**Reply by kRHYME7** [reply]
Rose pine

**Reply by Sofitt** [reply]
I cant find **any** wallpapers in repo. This is why i ask for file
